src/main/java/org/assertj/core/api/AbstractByteArrayAssert.java

```diff
@@ -220,6 +221,32 @@ public SELF contains(byte... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the given values, in any order.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 1,  2});
```

Comment:
Please ensure spaces around values:
```suggestion
   * assertThat(new byte[] { 1, 2, 3 }).contains(new Byte[] { 1,  2 });
```